cl_sii/libs/mm_utils.py

```diff
@@ -36,7 +38,7 @@ def validate_schema(self, data: dict, original_data: dict) -> None:
     # Original inspiration from
```

Comment:
In docstring of `validate_no_unexpected_input_fields()`, remove `class Meta: strict = True`.

> Setting the `strict` option on `class Meta` has no effect on **`Schema`** behavior.

Source: https://marshmallow.readthedocs.io/en/stable/upgrading.html#schemas-are-always-strict